src/index.js

```diff
@@ -15,11 +58,71 @@ function convert(object) {
   }, {})
 }
 
-function convertProperty(originalKey, value) {
+/**
+ * Converts a property and its value to the corresponding RTL key and value
+ * @param {String} originalKey the original property key
+ * @param {Number|String|Object} originalValue the original css property value
+ * @return {Object} the new {key, value} pair
+ */
+function convertProperty(originalKey, originalValue) {
   const key = getPropertyDoppelganger(originalKey)
+  const value = getValueDoppelganger(key, originalValue)
   return {key, value}
 }
 
+/**
+ * This gets the RTL version of the given property if it has a corresponding RTL property
+ * @param {String} property the name of the property
+ * @return {String} the name of the RTL property
+ */
 function getPropertyDoppelganger(property) {
   return propertiesToConvert[property] || property
 }
+
+/**
+ * This converts the given value to the RTL version of that value based on the key
+ * @param {String} key this is the key (note: this should be the RTL version of the originalKey)
+ * @param {String|Number|Object} originalValue the original css property value. If it's an object, then we'll convert that as well
+ * @return {String|Number|Object} the converted value
+ */
+function getValueDoppelganger(key, originalValue) {
+  /* eslint complexity:[2, 7] */ // let's try to keep the complexity down... If we have to do this much more, let's break this up
+  if (isObject(originalValue)) {
+    return convert(originalValue) // recurssion 🌀
+  }
+  const isNum = isNumber(originalValue)
+  const importantlessValue = isNum ? originalValue : originalValue.replace(/ !important.*?$/, '')
```

Comment:
True, I'll go ahead and merge as is because I'm on my phone. We can optimize later